config/routes.rb

```diff
@@ -7,6 +7,7 @@
   namespace :account do
     resources :companies
     resources :pages, only: %i[show index]
-    resources :tenants
+    resources :tenants, only: %i[show index]
+    resources :tenant, only: %i[show edit update]
```

Comment:
something weird here
mb try to use single-resource for tenant?
e.g.:
```
resource :tenant
```

more [there](http://guides.rubyonrails.org/routing.html#singular-resources)

Reply:
Done. But I renamed controller.